README.md

```diff
@@ -127,67 +179,85 @@ Ensure that the Louis Login Frontend React application is running, ideally on `l
 
 ---
 
-You can now interact with both the main Flask application and the client simulator to validate the entire authentication flow.
+You can now interact with both the main Quart application and the client simulator to validate the entire authentication flow.
 
 ## Running the app from dockerfile
 
-### Generate Server and Client Keys
+### 1. Generate Server, Client Keys, and Environment Files
 
 #### Prerequisites
 
 - OpenSSL installed on your machine (in WSL).
+- An Azure Communication Service connection string
+- An Azure MailFrom email address connected to the Azure Communication Service resource
 
 #### Steps
 
 1. Navigate to the project's root directory.
-2. Run the following script:
+2. Run the initialization script:
 
    ```bash
-   ./generate_keys.sh
+   ./init_project.sh <your-test-app-id>
    ```
 
-   This will generate four `.pem` files in a folder called `keys`:
+   This script will:
 
-   - `server_private_key.pem`
-   - `server_public_key.pem`
-   - `client_private_key.pem`
-   - `client_public_key.pem`
+   - Generate keys in a `keys` folder for both the server and the specified app id.
+   - Copy `.env.template` to `.env`.
+   - Copy `.env.tests.template` to `.env.tests`.
 
 #### Note
 
-- If the keys already exist, the script will overwrite them.
+- If the keys or environment files already exist, the script will overwrite them.
+- Logs are written to `init.log`.
```

Comment:
logging should use a generic Python logger facilities and be configurable from ENV. writing to a file is fine locally, but doesn't make sense server-side (where you need to output to stderr or use a cloud specific logger).

https://docs.python.org/3/library/logging.html

Reply:
this is only for the init-project.sh script. Only meant for local initialization of the project.